src/gil.rs

```diff
@@ -151,7 +151,7 @@ impl GILGuard {
                     "statically instead of with a shared library.\n\n",
                     "Please disable the `auto-initialize` feature, for example by entering the following ",
                     "in your cargo.toml:\n\n",
-                    "    pyo3 = { version = \"0.13.0\", default-features = false }\n\n",
+                    "    pyo3 = { version = \"0.13.1\", default-features = false }\n\n",
```

Comment:
Though I don't think we should fix this immediately, noting the precisely version here can be problematic when updating...

Reply:
Agreed, see #1373